src/main/scala/Generator.scala

```diff
@@ -15,7 +14,7 @@ trait GeneratorApp extends App with HasGeneratorUtilities {
     configProject = args(3),
     configs = args(4))
 
-  lazy val config = getConfig(names.fullConfigClasses)
+  lazy val config = getConfig(names)
```

Comment:
FWIW, the current rocket-chip master (freechipsproject/rocket-chip@dd075d3f2ab1e82bec53091e0027dc5827c92a24) still requires
```scala
lazy val config = getConfig(names.fullConfigClasses)
```
since `getConfig()` takes `Seq[String]`, not`ParsedInputNames`.